backend/package.json

```diff
@@ -21,28 +21,38 @@
     "doc": "compodoc -p tsconfig.doc.json"
   },
   "dependencies": {
+    "@img/sharp-win32-x64": "^0.33.0",
```

Comment:
But the `npm i` works perfectly fine with only `"sharp": "^0.33.0"`

Reply:
The images are correctly rotated on Linux with only "sharp": "^0.33.0". Let me know if it works on windows too

Reply:
> @liulw2022 Is it necessary? (it won't work on linux/macos)

I do need this when I'm running it  on my computer or it keeps getting an error, but I guess we can remove it? Because it's ultimately running on the server

Reply:
Yes but i don't know if Ekkart will test it on windows or linux...

Reply:
I just had a `optionalDependencies` so it should work on both linux and windows, could you try ?

Reply:
YES! Right now it perfectly works.